src/hardfault.c

```diff
@@ -14,6 +14,7 @@
 
 #include "hardfault.h"
 #include "util.h"
+#include <platform_config.h>
```

Comment:
This should go in a separate commit I guess?

Reply:
It is ;)